projects/plugins/boost/app/modules/optimizations/lcp/class-lcp.php

```diff
@@ -2,12 +2,17 @@
 
 namespace Automattic\Jetpack_Boost\Modules\Optimizations\Lcp;
 
+use Automattic\Jetpack\Schema\Schema;
+use Automattic\Jetpack\WP_JS_Data_Sync\Data_Sync;
 use Automattic\Jetpack_Boost\Contracts\Changes_Output_On_Activation;
 use Automattic\Jetpack_Boost\Contracts\Feature;
+use Automattic\Jetpack_Boost\Contracts\Has_Activate;
+use Automattic\Jetpack_Boost\Contracts\Has_Data_Sync;
+use Automattic\Jetpack_Boost\Contracts\Needs_To_Be_Ready;
 use Automattic\Jetpack_Boost\Contracts\Optimization;
 use Automattic\Jetpack_Boost\Lib\Output_Filter;
 
-class Lcp implements Feature, Changes_Output_On_Activation, Optimization {
+class Lcp implements Feature, Changes_Output_On_Activation, Optimization, Has_Activate, Needs_To_Be_Ready, Has_Data_Sync {
```

Comment:
Seems like it's not this PR that introduced it, but `Lcp` does not `Changes_Output_On_Activation`, it `Changes_Output_After_Activation`, because it needs to analyze first. 

```suggestion
class Lcp implements Feature, Changes_Output_After_Activation, Optimization, Has_Activate, Needs_To_Be_Ready, Has_Data_Sync {
```

Reply:
Nice spot, I've updated the LCP class to use `Changes_Output_After_Activation` instead now.